index.html

```diff
@@ -964,9 +964,9 @@ <h4><code>Timing-Allow-Origin</code> Response Header</h4>
       data-cite="FETCH#concept-request-window">window</a>'s
       <a data-cite="HTML/webappapis.html#concept-settings-object-origin">origin</a>,
       continue to the next <var>request</var>.</p></li>
-      <li><p>If the <a>Timing-Allow-Origin</a> header value list
-      does not contain a value which is byte-for-byte identical to
-      the <a
+      <li><p>If the <var>request</var>'s response' <a>Timing-Allow-Origin</a>
```

Comment:
Wouldn't it be "request's response's"? Maybe simplify to "If the TAO header value list of the request's response..."

Reply:
👍